src/workerd/io/compatibility-date.capnp

```diff
@@ -220,4 +220,9 @@ struct CompatibilityFlags @0x8f8c1b68151b6cef {
   # The Worker can also request specific compression settings by specifying a valid
   # Sec-WebSocket-Extensions header, or setting the header to the empty string to explicitly
   # request that no compression be used.
+
+  nodeCompatExperimental @21 :Bool
+        $compatEnableFlag("node_compat_experimental");
```

Comment:
Given that changing these are a breaking change we need to be careful about how we introduce them.

What version of the Node.js API would this be expected to enable?

Reply:
Also, the name really should be `nodejs` and not just `node`.

Perhaps we can consider encoding the Node.js LTS branch enabled by the flag into the name... for instance, Node.js 18.x is about to become LTS and should be our initial target for support here, so this can be:

```
nodejs18CompatExperimental
```

Or, since the 18.x LTS [code name](https://github.com/nodejs/Release/blob/main/CODENAMES.md) is `Hydrogen`:

```
nodejsHydrogenCompatExperimental
```

Reply:
Done `nodejs_18_compat_experimental`. I personally am not excited about code names (I'm still learning debian/ubuntu ones ))

Reply:
18 please 🙏